resources/definitions/fdmprinter.def.json

```diff
@@ -1258,6 +1259,23 @@
                     "limit_to_extruder": "wall_0_extruder_nr",
                     "settable_per_mesh": true
                 },
+                "z_seam_corner":
+                {
+                    "label": "Seam Corner Preference",
+                    "description": "Control whether corners on the model outline influence the position of the seam.",
```

Comment:
I'd like the description to contain some info on when a user should choose which option.
`Ignoring corners means the seam can end up on almost flat regions in the model. In order to hide the seam set the value to Inner Corners. In order to make the seam easily accessible set the value to Outer Corners. Choosing Any Corner means that an inner corner or an outer corner will be chosen - whichever is closest.`

Reply:
Actually i just talked to a colleague and we think you should rename the options to:
```
"z_seam_corner_none":  "Anywhere",
"z_seam_corner_inner": "Hide Seam",
"z_seam_corner_outer": "Expose Seam",
"z_seam_corner_any":   "Hide or Expose Seam"
```